cli/src/main/scala/co/uproot/abandon/App.scala

```diff
@@ -37,7 +37,7 @@ final class ReportWriter(settings: Settings, outFiles: Seq[String]) {
     }
 
     if (writesToScreen) {
-      s.foreach { print }
+      s.foreach { print(_) }
```

Comment:
@MiroslavCsonka Yes, they are the same. The problem comes from the compiler inferring `Any` when no placeholder is given:

```sbt
[warn] /opt/opensource/abandon/cli/src/main/scala/co/uproot/abandon/App.scala:40: Inferred type containing Any
[warn]       s.foreach { print }
```

With the placeholder value _ being passed, the warning is no longer given on compile. [Here is a link to](https://github.com/puffnfresh/wartremover#any) the wart rule that is triggering this.